drakrun/drakrun/draksetup.py

```diff
@@ -630,6 +660,33 @@ def postupgrade():
 
     detect_defaults()
 
+    install_info = InstallInfo.try_load()
+    # DRAKVUF Sandbox not installed. Nothing more to do
+    if not install_info:
+        return
```

Comment:
ig an accompanying `logging.error("drakvuf-sandbox not installed")` will be good. what do you say?

Reply:
+1

Reply:
I'll add a log message. It shouldn't be an error though.